Editor/src/gui/scenepanel.cpp

```diff
@@ -364,8 +364,90 @@ namespace Light
 				}
 				ImGui::EndCombo();
 			}
-
 			ImGui::Columns(1);
+
+			if(static_cast<int>(type)== 2)
```

Comment:
Please do comparison directly with enum and not integer for readability
```suggestion
			if(type == LightType::Spot)
```